keyboards/handwired/numbrero/readme.md

```diff
@@ -0,0 +1,22 @@
+#Numbrero
```

Comment:
Can you actually use our template for this? 

You can look at something like https://raw.githubusercontent.com/qmk/qmk_firmware/master/keyboards/1upkeyboards/1up60hse/readme.md to get an idea.

Reply:
Is there an *actual* template, e.g. something that perhaps should sit in a default file, since that link isn't a template.
I have no problem with using a template, just ..... you're not showing me one.... perhaps for addition on the 'to do' list, create a readme_template.md for people starting on creating these.

Reply:
Was not requested to do this either by noroadsleft or dreshna either, which is why I was not aware there was a templated format. Just sayin', perhaps consistency somewhere in maintainer/collaborator requests.